labman/db/tests/test_composition.py

```diff
@@ -115,6 +116,28 @@ def test_sample_composition_get_control_samples(self):
         self.assertEqual(SampleComposition.get_control_samples('TrOL'),
                          ['vibrio.positive.control'])
 
+    def test_sample_composition_specimen_id(self):
+        obs = SampleComposition(1).specimen_id
+        # returns the underlying id if no specimen_id_column is set
+        self.assertEqual(obs, '1.SKB1.640202')
+
```

Comment:
Might be nice to have the sort of blank test that is done after setting a specimen_id (copied below) also done here where one isn't set, to make sure we don't do anything later to screw up the handling of that case :)

            obs = SampleComposition(8).specimen_id
            self.assertEqual(obs, 'blank.21.H1')

Reply:
Great suggestion.